components/ca-bundle/development/user-ca-bundle.yaml

```diff
@@ -0,0 +1,79 @@
+---
+kind: ConfigMap
+metadata:
+  name: user-ca-bundle
+  namespace: openshift-config
+  annotations:
+    # https://github.com/argoproj/argo-cd/issues/5792#issuecomment-800940513
+    argocd.argoproj.io/compare-options: IgnoreExtraneous
+apiVersion: v1
+data:
+  ca-bundle.crt: |
```

Comment:
According to https://certs.corp.redhat.com/, it is recommended to be stored as if it was a secret... (See yellow warning in the download section). For the group sync, I did put them in vault, see https://github.com/redhat-appstudio/infra-deployments/blob/main/components/authentication/base/rh-idp/group-sync/external-secrets/mtls-ca-validators.yaml.

I know you did put them in the code already so though it is worth mentioning in case you were not aware of the recommendation, feel free to resolve this comment without changing anything.